cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=55)
+    description = serializers.CharField(max_length=255)
+    duration = serializers.IntegerField()
+
+    def create(self, validated_date):
```

Comment:
There is a typo in the method parameter `validated_date`. It should be `validated_data` to match the standard naming convention.